StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
Something to consider:
- What is the usage of PhysicalDiscNumber? Who will use it? Is it necessary for user? Which information will user need more?
- Should we make it organized? How about if we would like to add more information? Should we have a structure to contain them?